pmlc/dialect/pxa/analysis/uses.cc

```diff
@@ -73,7 +73,7 @@ IndirectAccessUsesIterator &IndirectAccessUsesIterator::operator++() {
 
 void IndirectAccessUsesIterator::skipNonAccess() {
   while (inner != IndirectUsesIterator()) {
-    if (isa<AffineLoadOp>(inner->getOwner()) ||
+    if (isa<pxa::AffineLoadOp>(inner->getOwner()) ||
```

Comment:
This may be overkill, but I hit a bug with namespace so I decided to make pxa vs. mlir namespace explicit everywhere for AffineLoad.

Reply:
I think explicit definitely better.